frontend/src/views/applications/ApplicationsHistoryView.vue

```diff
@@ -360,6 +360,24 @@ export default {
       })
       return applicationItems
     },
+
+    //TODO - Add Supp App PDF function will look very similar, but it will hit a seperate endpoint
+    //the supp app PDF's do not generate yet in Dynamics, so holding off on including that code
+    async downloadPDF(application) {
+      if (application.applicationType === 'OFM') {
+        const resp = await ApplicationService.getApplicationPDF(application.applicationId)
+
+        const link = document.createElement('a')
+        link.href = `data:application/pdf;base64,${resp}`
+        link.target = '_blank'
+        link.download = application.referenceNumber
+
+        // Simulate a click on the element <a>
+        document.body.appendChild(link)
+        link.click()
+        document.body.removeChild(link)
+      }
```

Comment:
Can we add a temporary alert if a Supplementary application is clicked? It might save some confusion during QA.

Reply:
Can do !